ChangeLog.md

```diff
@@ -278,6 +278,8 @@
   * Remove `string_from_2darray_sf()` from `freeformPythonLib/prairielearn.py` (Liz Livingston)
 
   * Remove `number` column from `question_tags` table; question tags are now sorted by `tags.number` (Nathan Walters).
+
+  * Remove support for `externalGradingOptions.files` in question `info.json` files (Nathan Walters).
```

Comment:
Maybe say `support for the deprecated XXX option`, so we don’t sound totally irresponsible? 😄